frame/lending/src/lib.rs

```diff
@@ -749,14 +754,18 @@ pub mod pallet {
 		/// liquidation.
 		/// - `origin` : Sender of this extrinsic.
 		/// - `market_id` : Market index from which `borrower` has taken borrow.
-		#[pallet::weight(<T as Config>::WeightInfo::liquidate(borrowers.len() as Weight))]
+		#[pallet::weight(<T as Config>::WeightInfo::liquidate(borrowers.len() as u32))]
```

Comment:
not sure. may be we should use usize?

Reply:
a auto-created file with weights (`runtime/dali/src/weights/lending.rs`) will have `fn liquidate(b: u32, ) -> Weight { ... }` independ on what we will use `u32` or `usize`. i.e. we will get a broken build with `usize`.